edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/ModuleToModuleResponseTimeout.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System;
+
+    // The role of this class is to help injecting a timeout value without activators
+    public class ModuleToModuleResponseTimeout
```

Comment:
nit: since TimeSpan is a struct, you can make it zero-cost abstraction ;)
```suggestion
    public struct ModuleToModuleResponseTimeout
```

Reply:
It is registered by autofac as an instance and it has problems with structures